stl/src/xlock.cpp

```diff
@@ -13,7 +13,7 @@
 
 _STD_BEGIN
 
-constexpr int _Max_lock = 8; // must be power of two
+constexpr int _Max_lock = 8; // must be power of two; TRANSITION, ABI: may be less now

```

Comment:
No change requested: It would actually be ABI-safe to reduce `_Max_lock` to `4`, assuming that nobody external was ever hijacking our internal APIs for their own nefarious purposes. I might do this in a followup PR. I noticed it too late to update this comment and it's not worth resetting testing.

(The remaining `_LOCK_LOCALE`, `_LOCK_STREAM`, and `_LOCK_DEBUG` constants must remain unchanged so that old-compiled code and new-compiled code can agree on what they're locking. We're just fortunate that they all happened to be less than `4`.)

Reply:
I don't know. As vNext is clearly not happening soon, we might want to keep some mutexes around for C++26 features or whatever.